.github/workflows/run-tests-on-release.yml

```
@@ -0,0 +1,253 @@
+      BACKUP_VER: ${{ steps.cloud_variables.outputs.BACKUP_VER }}
+      BACKUP_NAME: ${{ steps.cloud_variables.outputs.BACKUP_NAME }}
+      CHECK_ID: ${{ steps.create-check.outputs.CHECK_ID }}
+      FRAMEWORK: ${{ steps.check-framework.outputs.result }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - id: check-framework
+        uses: actions/github-script@v7
+        env:
+          VERSION: "${{inputs.VERSION}}"
+        with:
+          script: |
+            const { VERSION } = process.env
+            const versionWithoutDot = VERSION.replaceAll(/\./g, "")
+            if(versionWithoutDot >= 319){
+              return "playwright"
+            }else{
+              return "cypress"
+            }
+
+      - name: Create check if release PR exists
+        if: steps.check-framework.outputs.result == '"playwright"'
+        id: create-check
+        uses: ./.github/actions/checks/create-pull-request-check
+        with:
+          PR_HEAD: "changeset-release/${{env.MINOR_VERSION}}"
+          CHECK_NAME: "Automation_tests_run"
+          STATUS: "in_progress"
+          TITLE: "Automation tests run"
+          DETAILS_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+      - name: Generate variables
+        id: cloud_variables
+        uses: ./.github/actions/prepare-tests-variables
+        with:
+          CLI_TOKEN: ${{ secrets.STAGING_TOKEN }}
+          MODE: "from-version"
+          VERSION: ${{ env.VERSION }}
+
+      - name: Prepare instances
+        if: steps.check-framework.outputs.result == '"playwright"' 
+        id: pool_variables
+        uses: ./.github/actions/prepare-instance
+        with:
+          STRATEGY: reload
+          CLI_TOKEN: ${{ secrets.STAGING_TOKEN }}
+          BASE_URL: ${{ steps.cloud_variables.outputs.BASE_URL }}
+          API_URL: ${{ steps.cloud_variables.outputs.API_URL }}
+          POOL_NAME: ${{ steps.cloud_variables.outputs.POOL_NAME }}
+          POOL_INSTANCE: ${{ steps.cloud_variables.outputs.POOL_INSTANCE }}
+          BACKUP_ID: ${{ steps.cloud_variables.outputs.BACKUP_ID }}
+
+      - uses: ./.github/actions/testmo/testmo-init
+        if: steps.check-framework.outputs.result == '"playwright"' 
+        with:
+          testmoUrl: ${{ secrets.TESTMO_URL }}
+          testmoToken: ${{ secrets.TESTMO_TOKEN }}
+          testmoRunName: "Playwright run ${{ env.CUSTOM_VERSION }}"
+        id: init-testmo
+
+  run-cy-tests:
+    runs-on: ubuntu-22.04
+    needs: "add-check-and-prepare-instance"
+    if: ${{ needs.add-check-and-prepare-instance.outputs.FRAMEWORK == '"cypress"' }}
+    steps:
+      - name: Trigger dashboard cypress automation tests
+        env: 
+          VERSION: ${{inputs.CUSTOM_VERSION}}
+        run: |
+          export GITHUB_TOKEN=$( \
+            curl --request GET --url ${{ secrets.VAULT_URL}} --header "Authorization: JWT ${{ secrets.VAULT_JWT }}" | jq -r .token \
+          )
```

If the `curl` command fails, it will not exit with non-zero status when doing `export foo=$(false)` thus the workflow will not be marked as failed.

Example:

```
$ bash -ec 'foo=$(false); export foo'; echo $?
1
$ bash -ec 'export foo=$(false)'; echo $?
0
```

```suggestion
          GITHUB_TOKEN=$( \
            curl --request GET --url ${{ secrets.VAULT_URL}} --header "Authorization: JWT ${{ secrets.VAULT_JWT }}" | jq -r .token \
          )
          export "$GITHUB_TOKEN"
```